internal/driver/glfw/driver.go

```diff
@@ -113,6 +114,16 @@ func (d *gLDriver) windowList() []fyne.Window {
 	return d.windows
 }
 
+func (d *gLDriver) initFailed(msg string, err error) {
+	fyne.LogError(msg, err)
+
+	if running() {
+		d.Quit()
+	} else {
+		os.Exit(1)
```

Comment:
It would mean that if the creation of any window in the application runtime fails, the whole app will quit? Is that intended?

Reply:
> When a window cannot be opened it basically means the graphics driver connection has failed or we are out of memory - both are pretty much irrecoverable.

That's a good point, I don't know what other things could fail. 
I was just thinking in the case that an application could try to create a new window, and it failed but not with a severe error, the application could continue working and just ignore that error and don't create the window. In that sense, the driver should quit the app if the creation of the `master` window fails.

However, if an error in window creation is always a severe error, then I think it is fine :)